book/src/bridge/attributes.md

```diff
@@ -36,4 +36,10 @@ For [`#[qproperty]`](./extern_rustqt.md#properties), a CXX or Rust name can be p
 
 > **⚠️ Deprecation warning**:
 > CXX-Qt <0.6 did automatic case conversion if no `#[cxx_name = "..."]` or `#[rust_name = "..."]` is specified.
-> Starting with CXX-Qt 0.7, this is no longer the case!
+> Starting with CXX-Qt 0.7, this is no longer the case! Automatic case conversion will be opt-in instead.
+
+### Automatic case conversion
+
+The `#[auto_cxx_name]` and `#[auto_rust_name]` attributes can be used to automatically rename cxx and rust names.
+These are placed at a block level on extern "RustQt" or "C++Qt" blocks, and will automatically case convert the items inside, unless they specify either rust or cxx name.
```

Comment:
I would write the block names fully in code blocks so
```
These are placed at a block level on `extern "RustQt"` or `extern "C++Qt"` blocks
```

Then I would change the last part to
```
unless they specify either a `rust_name` or `cxx_name`.
```